frontend/src/assets/i18n/de.json

```diff
@@ -65,6 +65,42 @@
       }
     }
   },
+  "CONFIRMATION": {
+    "DRAFT_CREATE": {
+      "TITLE": "Check-in im Draft-Status",
+      "TEXT": "Dein Objective befindet sich noch im DRAFT Status. Möchtest du das Check-in trotzdem erfassen?"
+    },
+    "RELEASE": {
+      "TITLE": "Objective veröffentlichen",
+      "TEXT": "Soll dieses Objective veröffentlicht werden?"
+    },
+    "TO_DRAFT": {
+      "TITLE": "Objective als Draft speichern",
+      "TEXT": "Soll dieses Objective als Draft gespeichert werden?"
+    },
+    "DELETE": {
+      "ACTION":{
+        "TITLE": "Löschen bestätigen",
+        "TEXT": "Möchtest du die Action wirklich löschen?"
+      },
+      "TEAM":{
+        "TITLE": "Team löschen",
+        "TEXT": "Möchtest du das Team '{{team}}' wirklich löschen? Zugehörige Objectives werden dadurch in allen Quartalen ebenfalls gelöscht!"
+      },
+      "OBJECTIVE":{
+        "TITLE": "Objective löschen",
+        "TEXT": "Möchtest du dieses Objective wirklich löschen? Zugehörige Key Results werden dadurch ebenfalls gelöscht!"
+      },
+      "KEYRESULT":{
+        "TITLE": "Key Result löschen",
+        "TEXT": "Möchtest du dieses Key Result wirklich löschen? Zugehörige Check-ins werden dadurch ebenfalls gelöscht!"
+      },
+      "USER_FROM_TEAM":{
+        "TITLE": "Mitglied entfernen",
+        "TEXT": "Möchtest du {{user}} wirklich aus dem Team '{{team}}' entfernen?"
+      }
+    }
+  },
```

Comment:
Der ganze `CONFIRMATION`-Key is dupliziert, nehme an das ist ein Fehler durch den Rebase?

Reply:
ist auch ein rebase Fehler